server/src/main/java/io/crate/metadata/SimpleReference.java

```diff
@@ -210,6 +267,7 @@ public Map<String, Object> toMapping() {
         Map<String, Object> mapping = new HashMap<>();
         mapping.put("type", DataTypes.esMappingNameFrom(innerType.id()));
         mapping.put("position", position);
+        mapping.put("oid", oid);
```

Comment:
just naming: should I rename it to `column_oid` to align with metadata counter?
currently have `oid` here and in field mappers as by the context it's clear it's column specific while in metadata it's not.